controllers/snow_machineconfig_controller.go

```diff
@@ -81,9 +80,6 @@ func (r *SnowMachineConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 func (r *SnowMachineConfigReconciler) reconcile(ctx context.Context, snowMachineConfig *anywherev1.SnowMachineConfig) (_ ctrl.Result, reterr error) {
 	var allErrs []error
-	if err := r.validator.ValidateMachineDeviceIPs(ctx, snowMachineConfig); err != nil {
```

Comment:
Are we doing this because now we check for this in the other two validate functions?

Reply:
yes, also the new ValidateDeviceIsUnlocked should already checked this too.